locales-pending/fix.ftl

```diff
@@ -0,0 +1,63 @@
+fix-flow-data-broker-profiles-manual-remove-review-profiles-headline = Review & remove your profiles
+fix-flow-data-broker-profiles-manual-remove-button-remove-for-me = Remove them for me
+fix-flow-data-broker-profiles-manual-remove-button-skip = Skip for now 
+fix-flow-data-broker-profiles-automatic-remove-headline = We’ll auto-remove these and guide you through exposures that need manual fixes
+# Variables:
+# $data_broker_count is the number of data brokers scanned monthly (As of adding this, it is 190). This number will always be plural.
+fix-flow-data-broker-profiles-automatic-remove-subheadline = Removing these stops the sale of your personal info on { $data_broker_count } sites. Upgrade to Monitor Premium and you’ll get:
+# Variables:
+# $percent is the percentage you can save subscribing to an annual/yearly plan
+fix-flow-data-broker-profiles-automatic-remove-save-percent = Save { $percent }% with yearly plan
+fix-flow-data-broker-profiles-automatic-remove-features-select-plan-headline = Premium Protection
+fix-flow-data-broker-profiles-automatic-remove-features-select-plan-toggle-yearly =  Yearly
+fix-flow-data-broker-profiles-automatic-remove-features-select-plan-toggle-monthly =  Monthly
+fix-flow-data-broker-profiles-automatic-remove-features-select-plan-monthly-frequency = Billed monthly
+fix-flow-data-broker-profiles-automatic-remove-features-select-plan-monthly-button = Select monthly plan
+fix-flow-data-broker-profiles-automatic-remove-features-select-plan-yearly-frequency = Billed yearly
+fix-flow-data-broker-profiles-automatic-remove-features-select-plan-yearly-button = Select yearly plan
+fix-flow-data-broker-profiles-automatic-remove-features-headline = Features:
+fix-flow-data-broker-profiles-automatic-remove-features-remove-personal-info = Automatic removal of personal info from sites that are selling it
+fix-flow-data-broker-profiles-automatic-remove-features-guided-experience = Guided experience through high risk data breaches that require manual steps
+fix-flow-data-broker-profiles-automatic-remove-features-continuous-monitoring = Continuous monitoring for new exposures
+fix-flow-data-broker-profiles-automatic-remove-features-breach-alerts =  Alerts when your data has been breached
+# Variables:
+# $data_broker_count is the number of data brokers scanned monthly (As of adding this, it is 190). This number will always be plural.
+fix-flow-data-broker-profiles-automatic-remove-features-monthly-scan = Monthly scan of { $data_broker_count } data broker sites that may be selling your personal info
+# Variables:
+# $price is the price of a monthly plan. "/mo" is short for "monthly"
+fix-flow-data-broker-profiles-automatic-remove-features-price = ${ $price }/mo
```

Comment:
I had the same question. We're displaying the "per month" price in both contexts, rather than showing the _per year_ price for the annual plan.

Leaving as is.